jinahub/indexers/searcher/FaissSearcher/README.md

```diff
@@ -1,63 +1,63 @@
 # FaissSearcher
 
-**FaissSearcher** is a Faiss-powered vector Searcher.
-
-Faiss is a library for efficient similarity search and clustering of dense vectors. It contains algorithms that search in sets of vectors of any size, up to ones that possibly do not fit in RAM. It also contains supporting code for evaluation and parameter tuning. Faiss is written in C++ with complete wrappers for Python/numpy. Some of the most useful algorithms are implemented on the GPU. It is developed by Facebook AI Research.
-
-
+**FaissSearcher** wraps Faiss-powered vector Searcher.
 
+**FaissSearch** is defined as a vector searcher,
+These usually implement a form of similarity search,
+based on the embeddings created by the encoders you have chosen in your Flow.
+Vector searcher is meant to be used together with a **Storage**.
+To understand Jina's storage-search workflow,
+please read the documentation [here](https://docs.jina.ai/advanced/experimental/indexers/).
 
+[Faiss](https://github.com/facebookresearch/faiss) is a library for efficient similarity search and clustering of dense vectors.
+It contains algorithms that search in sets of vectors of any size, up to ones that possibly do not fit in RAM.
+It also contains supporting code for evaluation and parameter tuning.
+Faiss is written in C++ with complete wrappers for Python/numpy.
+Some of the most useful algorithms are implemented on the GPU.
+It is developed by Facebook AI Research.
 
 
 ## Usage
 
-Check [tests](tests) for an example on how to use it.
-
 ### Loading data
 
-Since this is a "Searcher"-type Executor, it does not _index_ new data. Rather they are write-once classes, which take as data source a `dump_path`. 
-
-This can be provided in different ways:
-
-- in the YAML definition
-  
-```yaml
-jtype: FaissSearcher
-with:
-    dump_path: /tmp/your_dump_location
-...
-```
-
-- from the `Flow.rolling_update` method. See [docs](https://docs.jina.ai/fundamentals/executor/indexers/).
+Since this is a "Searcher"-type Executor, it does not _index_ new data.
```

Comment:
Can you just link to the documentation? This is going to change a bit once we release the load from PSQL, and we'd rather not have to edit every file, but only one "source of truth"